src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -730,6 +731,83 @@ public async Task PostImportAsync()
             }
         }
 
+        private void IgnoreDatabaseItemTypesIfCurrentVersionIsOlderThenClientVersion(DocumentsOperationContext context, ref BlittableJsonReaderObject blittableJson)
+        {
+            if (blittableJson.TryGet(nameof(DatabaseSmugglerOptions.OperateOnTypes), out string operateOnTypes) == false &&
+                blittableJson.TryGet(nameof(DatabaseSmugglerOptions.OperateOnTypes), out int _) == false)
+                throw new InvalidOperationException($"Invalid value for {nameof(DatabaseSmugglerOptions.OperateOnTypes)}, must be non-empty 'string' or 'int'");
```

Comment:
let's wait for the tests, but this added non-existing requirement I think? previously there was no need to provide it in JSON. IMO, probably, you should not do the check if it is empty.

Reply:
done